native/src/widget/slider.rs

```diff
@@ -171,6 +187,9 @@ where
                     }
                 }
                 mouse::Event::ButtonReleased(mouse::Button::Left) => {
```

Comment:
I just noticed we may need to check `self.state.is_dragging` is `true` in this branch.

Reply:
Good catch!